web/src/components/navbar/NavBar.test.tsx

```diff
@@ -69,6 +69,68 @@ describe("<NavBar />", () => {
     });
   });
 
+  describe("navbar - used when user is onboarding", () => {
+    beforeEach(() => {
+      useMockUserData({});
+      useMockRouter({ pathname: "/onboarding" });
+    });
+
+    describe("desktop version", () => {
+      const renderDesktopNav = () => {
+        setLargeScreen(true);
+        render(
+          withAuth(
+            <NavBar landingPage={false} task={undefined} sidebarPageLayout={false} operateReferences={{}} />,
+            { isAuthenticated: IsAuthenticated.FALSE }
+          )
+        );
+      };
+
+      it("renders the log in button", async () => {
+        renderDesktopNav();
+        expect(screen.getByText(Config.navigationDefaults.logInButton)).toBeInTheDocument();
+      });
+
+      it("doesn't render register button when in onboarding", () => {
+        renderDesktopNav();
+        expect(screen.queryByTestId("registration-button")).not.toBeInTheDocument();
+      });
+
+      it("doesn't render a menu to be opened when in onboarding", async () => {
```

Comment:
in this test name and the one before it, the "when in onboarding" piece is redundant because it already says that in the `describe`